tests/network-tests/src/fixtures/content/activeVideoCounters.ts

```diff
@@ -122,21 +122,16 @@ export class ActiveVideoCountersFixture extends BaseQueryNodeFixture {
     in Query node.
   */
   private async assertCounterMatch(
-    entityName: 'channels' | 'channelCategories' | 'videoCategories',
+    entityName: 'channel' | 'channelCategory' | 'videoCategory',
     entityId: number,
     expectedCount: number
   ) {
-    const getterName = `get${entityName[0].toUpperCase()}${entityName.slice(1)}` as
-      | 'getChannels'
-      | 'getChannelCategories'
-      | 'getVideoCategories'
+    const getterName = `${entityName}ById` as 'channelById' | 'channelCategoryById' | 'videoCategoryById'
     await this.query.tryQueryWithTimeout(
-      () => this.query[getterName](),
+      () => this.query[getterName](entityId.toString()),
       (tmpEntity) => {
-        const entities = (tmpEntity as any).data[entityName]
-        assert(entities.length > 0) // some entities were loaded
-
-        const entity = entities.find((item: any) => item.id === entityId.toString())
+        assert(tmpEntity)
+        const entity = tmpEntity as Exclude<typeof tmpEntity, null>
```

Comment:
`Utils.assert` provides typesafety support for this case

Reply:
`Utils.assert` is used here now, thx.